plugins/plugin-template/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
We should add content to the README.md file to instruct the plugin creator to add details about the plugin. we can take example from the existing plugins.